# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 8bee937821e3
# Full Hash: 8bee937821e3725b922352a0493f53b5e431c3d0
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-05-25 09:34:31
# Regressor Bug: 1586236
# File Overlap Count: 1
# Description:
#   Bug 1711610 - Should not trigger the memory pressure event if the available commit space is not low.  r=gsvelto
#   
#   After bug 1586236, we use the memory resource notification object to detect a low
#   memory situation on Windows, which is signaled when the available physical memory
#   is low.  If the available physical memory is low, however, it's possible that there
# ==============================================================================

diff -r e24c277bcb6c -r 8bee937821e3 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon May 24 21:07:40 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon May 24 21:29:46 2021 +0000
@@ -1121,6 +1121,16 @@
   value: 2000
   mirror: always
 
+#ifdef XP_WIN
+  # Notify TabUnloader or send the memory pressure if the memory resource
+  # notification is signaled AND the available commit space is lower than
+  # this value.
+-   name: browser.low_commit_space_threshold_mb
+    type: RelaxedAtomicUint32
+    value: 200
+    mirror: always
+#endif
+
 # Render animations and videos as a solid color
 - name: browser.measurement.render_anims_and_video_solid
   type: RelaxedAtomicBool